src/core/parameters.js

```diff
@@ -0,0 +1,175 @@
+const endpoints = {
+  setup: {
```

Comment:
@ManuelFernando can we move those configs into individual files? this way when we modify the params, we can easily adjust.